cypress/e2e/homePage-spec.cy.js

```diff
@@ -1,5 +1,78 @@
-describe('template spec', () => {
-  it('passes', () => {
-    cy.visit('https://example.cypress.io')
+describe('home page', () => {
+    beforeEach(() => {
+      cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies', {
+        statusCode: 200,
+        body: {
```

Comment:
A fixture here would improve readability by abstacting this mock data into a separate file, saving you 15 lines of mock data code in your test.